client/src/components/common/boost/BoostConfigurator.tsx

```diff
@@ -23,17 +25,49 @@ export default function BoostConfigurator({
   onDeleteConfig,
   onUploadConfig,
 }: BoostConfiguratorProps) {
+  const fileInput = createRef<HTMLInputElement>();
+  const [configType, setConfigType] = useState<BoostConfigType>('all');
+
+  // Function to click the hidden file input button (this is a work around to avoid the ugly
+  // UI of the default input file button)
+  const handleClick = (type: BoostConfigType) => {
```

Comment:
Nice fix here! I would name the function something more descriptive, like `handleClickUpload`